homeassistant/components/comelit/config_flow.py

```diff
@@ -39,7 +46,11 @@ async def validate_input(
 ) -> dict[str, str]:
     """Validate the user input allows us to connect."""
 
-    api = ComeliteSerialBridgeApi(data[CONF_HOST], data[CONF_PORT], data[CONF_PIN])
+    api: ComelitCommonApi
+    if data.get(CONF_TYPE, BRIDGE) == BRIDGE:
```

Comment:
```suggestion
    if data[CONF_TYPE] == BRIDGE:
```

Reply:
Current config entries don't have the "CONF_TYPE" so we need to guard in order to handle.